grails-app/controllers/org/transmartproject/rest/ExportController.groovy

```diff
@@ -29,29 +29,27 @@ class ExportController {
      *  Returns datatypes and patient number of given concepts.
      *
      */
-    def datatypes(){
-        def jsonSlurper = new JsonSlurper()
+    def datatypes() throws NoSuchResourceException {
         if (!(params.containsKey('concepts'))){
-            throw new NoSuchElementException(
-                    "No parameter named concepts."
-            )
+            throw new NoSuchResourceException("No parameter named concepts was given.")
+        }
+        if (params.get('concepts') == "") {
```

Comment:
I tested it with "/v1/export/datatypes?concepts=" and "/v1/export/datatypes?concepts". Both return "Parameter concepts has no value."